tests/unittest/_torch/cute_dsl/test_kimi_k3_kda_ptx_patch.py

```
@@ -0,0 +1,168 @@
+import subprocess
+import sys
+from pathlib import Path
+
+import pytest
+
+try:
+    import cutlass  # noqa: F401
+    import flashinfer.gdn_kernels  # noqa: F401
+
+    _KERNEL_DEPS_AVAILABLE = True
+except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)